src/main/java/eu/deltasource/internship/hotelmanagementsystem/UnavailableRooms.java

```diff
@@ -0,0 +1,10 @@
+package eu.deltasource.internship.hotelmanagementsystem;
+
+
+public class UnavailableRooms extends RuntimeException {
```

Comment:
Mentioned that the name should be suffixed with exception for example "NoAvailableRoomsException"